### ISSUE TITLE
[Library] Chrome Audit

### ISSUE THREAD

**Issue by richardhand** [adobe/aem-core-wcm-components#484]
A Chrome audit of the Component Library reports a number of issues related to Accessibility / SEO etc. that should be addressed.
